rust/bindings/lib.rs

```diff
@@ -3,13 +3,20 @@
 //! Bindings.
 //!
 //! Imports the generated bindings by `bindgen`.
+//!
+//! This crate may not be directly used. If you need a kernel C API that is
+//! not ported or wrapped in the kernel crate, then do so first instead of
```

Comment:
I typically use monospace when talking about the crate, to distinguish it from the kernel itself in e.g. "kernel C API" above.

```suggestion
//! not ported or wrapped in the `kernel` crate, then do so first instead of
```